plugins/modules/appliance_access_consolecli.py

```diff
@@ -102,8 +103,8 @@
     "set": {"query": {}, "body": {"enabled": "enabled"}, "path": {}}
 }  # pylint: disable=line-too-long
 
-import json
-import socket
+import json  # pylint: disable=unused-import
```

Comment:
I would rather prefer the generator to import only required modules

Reply:
Agree!

Reply:
We have a single header template file, which is rendered into all the module files. Some modules use the imported libraries and some do not. Instead of skipping the pylint in the individual module files, I think adding the pylint disable in the template file is less time-consuming. What are your thoughts.?